tests/manageability/tests/container_port_name_format.go

```diff
@@ -10,24 +12,58 @@ import (
 	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
 )
 
-var _ = Describe("manageability-container-port-name", Serial, func() {
+var _ = Describe("manageability-container-port-name", func() {
+	var randomNamespace string
+	var origReportDir string
+	var origTnfConfigDir string
 
 	BeforeEach(func() {
-		By("Clean namespace before each test")
-		err := namespaces.Clean(tsparams.ManageabilityNamespace, globalhelper.GetAPIClient())
+		randomNamespace = tsparams.ManageabilityNamespace + "-" + globalhelper.GenerateRandomString(10)
```

Comment:
Shouldn't we create a function to avoid duplication in beforeEach and AfterEach ?